src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -645,12 +645,24 @@ def test_blink_device_light(self, _run_cephadm, on_bool, fault_ident, cephadm_mo
     def test_blink_device_light_custom(self, _run_cephadm, cephadm_module):
         _run_cephadm.return_value = '{}', '', 0
         with with_host(cephadm_module, 'test'):
-            cephadm_module.set_store('lsmcli_blink_lights_cmd', 'echo hello')
+            cephadm_module.set_store('blink_device_light_cmd', 'echo hello')
```

Comment:
Users can replace "echo hello" by "cd / && rm -rf" without problems.
Demonstration of how dangerous can be the implementation of this command. ( templates should be used for config settings ,,... never for commands, imho)

Reply:
You're right, its potential a risk, but this PR has not introduced that feature, it only adds some missing behavior. And there is the need to allow the user to modify the command to switch the LEDs on/off.

Reply:
I have realized of the "command in template" problem looking at this modification. This is why i have commented here.

I think that the way to solve it is:

1. Do not use an external template, in this case is just a command, we can store it as an internal variable.

2. Customize by host:
Here is where i see the real problem. In my view we should in any case examine the new platform where we want to use this feature and include the command in our "supported" blinking lights commands set.
We have the gather fact feature to examine the hardware an to know when we need to use one command or another, and thus avoiding the user putting different commands for different hosts in the mon store.